projects/igniteui-angular/src/lib/simple-combo/simple-combo.component.spec.ts

```diff
@@ -554,6 +914,222 @@ describe('IgxSimpleCombo', () => {
             fixture.detectChanges();
             expect(document.activeElement).toEqual(addItemButton.nativeElement);
         }));
+
+        it('should close when an item is clicked on', fakeAsync(() => {
+            spyOn(combo, 'close').and.callThrough();
+            combo.open();
+            fixture.detectChanges();
+            const item1 = fixture.debugElement.query(By.css(`.${CSS_CLASS_DROPDOWNLISTITEM}`));
+            expect(item1).toBeDefined();
+
+            UIInteractions.simulateClickEvent(item1.nativeElement);
+            fixture.detectChanges();
+
+            expect(combo.close).toHaveBeenCalledTimes(1);
+        }));
+
+        it('should scroll to top when opened and there is no selection', fakeAsync(() => {
+            combo.deselect();
+            fixture.detectChanges();
+
+            spyOn(combo, 'onClick').and.callThrough();
+            spyOn((combo as any).virtDir, 'scrollTo').and.callThrough();
+
+            const toggleButton = fixture.debugElement.query(By.directive(IgxIconComponent));
+            expect(toggleButton).toBeDefined();
+
+            UIInteractions.simulateClickEvent(toggleButton.nativeElement);
+            fixture.detectChanges();
+
+            expect(combo.onClick).toHaveBeenCalledTimes(1);
+            expect((combo as any).virtDir.scrollTo).toHaveBeenCalledWith(0);
+        }));
+
+        it('should close the dropdown with Alt + ArrowUp', fakeAsync(() => {
+            combo.open();
+            fixture.detectChanges();
+
+            spyOn(combo, 'close').and.callThrough();
+
+            UIInteractions.triggerKeyDownEvtUponElem('ArrowDown', input.nativeElement);
```

Comment:
Use triggerEventHandlerKeyDown on the inputq to which the event handler is attached instead of triggerEventHandlerKeyDown

const CSS_CLASS_COMBO_INPUT = 'igx-input-group__input';
 const inputEl=fixture.debugElement.query(By.css(`.${CSS_CLASS_COMBO_INPUT}`));
 UIInteractions.triggerEventHandlerKeyDown('ArrowDown', inputEl);
 tick();
 fixture.detectChanges();